cfgov/unprocessed/css/forms.less

```diff
@@ -650,21 +441,315 @@
     }
 });
 
+
+/* topdoc
+  name: Checkboxes and radio buttons
+    family: cf-forms
+    notes:
+      - "CSS for checkboxes and radio buttons requires following a specific HTML structure for each type of input, otherwise the styles will not be properly applied."
+      - "You MUST wrap your input.cf-input in a label."
+      - "Remember to use 'for' and 'id' attributes so that selected/checked styles function properly."
+      - "Remember to use the same 'name' attribute for each set of radio buttons."
+    patterns:
+      - name: Checkbox
+        markup: |
+          <label class="cf-input_label">
+              <input class="cf-input cf-input_checkbox" type="checkbox" name="option1" value="Option 1">
```

Comment:
This example code looks like it's outdated.. at least according to the updated pages (e.g. /learn-page/)